website/docs/r/kubernetes_auth_backend_role.html.md

```diff
@@ -41,6 +42,8 @@ The following arguments are supported:
 
 * `backend` - (Optional) Unique name of the kubernetes backend to configure.
 
+* `audience` - (Optional) Audience claim to verify in the JWT.
```

Comment:
Per my comment above, treating this as plural—as a set—would be more useful.

Reply:
From our convo above, I think it should be fine as-is. Thank you for looking at this!